spec/std/isa/csr/Smaia/mtopei.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Qualcomm Technologies, Inc. and/or its subsidiaries.
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+
+# yaml-language-server: $schema=../../../schemas/csr_schema.json
+
+$schema: "csr_schema.json#"
+kind: csr
+name: mtopei
+long_name: Machine Top External Interrupt
+address: 0x35C
+writable: true
+priv_mode: M
+length: MXLEN
+definedBy: Smaia
+description: |
+  The `mtopei` register provides information about the highest-priority
+  external interrupt that is currently enabled and pending for M-mode.
+  
+  This register is used in conjunction with the IMSIC (Incoming Message-Signaled
+  Interrupt Controller) to efficiently handle external interrupts.
+  
+  Reading this register returns the interrupt identity of the highest-priority
+  pending and enabled external interrupt. Writing to this register can be used
+  to claim and clear the interrupt.
+  
+  The format of this register depends on the IMSIC implementation and the
+  number of supported interrupt identities.
```

Comment:
Is this true? Where did this text originate?

Reply:
Closing in the absence of clarifying information. The text in question is no longer present.